src/borg/archiver.py

```diff
@@ -599,12 +600,16 @@ def chunkit(chunker_name, *args, **kwargs):
                 key_256, key_256, iv=key_128, header_len=1, aad_offset=1).encrypt(random_10M, header=b'X')),
             ("aes-256-ctr-blake2b", lambda: AES256_CTR_BLAKE2b(
                 key_256*4, key_256, iv=key_128, header_len=1, aad_offset=1).encrypt(random_10M, header=b'X')),
-            ("aes-256-ocb", lambda: AES256_OCB(
-                None, key_256, iv=key_96, header_len=1, aad_offset=1).encrypt(random_10M, header=b'X')),
-            ("chacha20-poly1305", lambda: CHACHA20_POLY1305(
-                None, key_256, iv=key_96, header_len=1, aad_offset=1).encrypt(random_10M, header=b'X')),
         ]:
             print(f"{spec:<24} {size:<10} {timeit(func, number=100):.3f}s")
+        if not is_libressl:
+            for spec, func in [
+                ("aes-256-ocb", lambda: AES256_OCB(
+                    None, key_256, iv=key_96, header_len=1, aad_offset=1).encrypt(random_10M, header=b'X')),
+                ("chacha20-poly1305", lambda: CHACHA20_POLY1305(
+                    None, key_256, iv=key_96, header_len=1, aad_offset=1).encrypt(random_10M, header=b'X')),
+            ]:
+                print(f"{spec:<24} {size:<10} {timeit(func, number=100):.3f}s")
```

Comment:
Oops, yeah, forgot about libressl's reduced capabilities... - Thanks for fixing!

Hmm, maybe just extend the tests lists? Like:
```
tests = [...works for both...]
if not is_libressl:
    tests.extend([...only for openssl...])
for spec, func in tests: ...
```